modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/JSON.mustache

```diff
@@ -37,6 +37,7 @@ import java.text.ParsePosition;
 import java.time.LocalDate;
 import java.time.OffsetDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.Base64;
```

Comment:
Okhttp has a Base64 class that can be used instead of the Java8 one. This way android users can enjoy the fix.